src/state/apis/swapper/swapperApi.ts

```diff
@@ -21,36 +22,15 @@ import { selectFeatureFlags } from 'state/slices/preferencesSlice/selectors'
 import { selectInputSellAsset } from 'state/slices/tradeInputSlice/selectors'
 
 import { BASE_RTK_CREATE_API_CONFIG } from '../const'
-import { apiErrorHandler } from '../utils'
-import { isTradingActive } from './helpers'
 import { validateTradeQuote } from './helpers/validateTradeQuote'
 
-const getIsTradingActiveErrorHandler = apiErrorHandler(
-  'getIsTradingActiveApi: error getting trading status',
-)
-
 export const GET_TRADE_QUOTE_POLLING_INTERVAL = 20_000
 export const swapperApiBase = createApi({
```

Comment:
We may want to remove this guy which now doesn't contain any query - @woodenfurniture to confirm

Reply:
correct yeah, or rather, remove the `injectEndpoints` below and have just the one swapperApi

Reply:
gm 54d55b0fcf132b7c5ca12a7391525c2a935692c6